packages/frontend/src/view/pages/forcedactions/ForcedWithdrawPage.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+
+import { Button } from '../../components/common/Button'
+import { Card } from '../../components/common/Card'
+import { Link } from '../../components/common/Link'
+import { OrderedList } from '../../components/common/OrderedList'
+import { Page } from '../../components/common/page/Page'
+import { ForcedActionCard } from '../../components/forcedaction/ForcedActionCard'
+import { AmountInput } from '../../components/forcedaction/form/AmountInput'
+import { FormId } from '../../components/forcedaction/form/ids'
+import { reactToHtml } from '../../reactToHtml'
+import {
+  ForcedActionFormProps,
+  serializeForcedActionsFormProps,
+} from './ForcedActionFormProps'
+
+function ForcedWithdrawPage(props: ForcedActionFormProps) {
+  const instructions = [
+    <span>
+      Using this form you request a withdrawal of your funds. (Perpetual only)
+      You can only withdraw USDC, so to fully exit all funds you should first
+      get rid of your synthetic assets by using the close functionality.
+      (Perpetual only) This is achieved through a mechanism called forced
+      withdrawals, <Link href="https://google.com">link to docs</Link>.
```

Comment:
Don't use random urls. If you don't have a link 1) ask 2) if there is none use `#`. In this case the link is `https://docs.starkware.co/starkex/perpetual/perpetual-trading-forced-withdrawal-and-forced-trade.html#forced_withdrawal`